### PR TITLE
Update demo_file.java

### PR DESCRIPTION
**Pull request by hemanthReddy125**
hemanthreddy739 check it once, i'm not confident on this code

### CODE REVIEW

File: demo_file.java [hunk 1]
```diff
@@ -8,3 +8,9 @@ public static void main(String args[]){
 
 
 // it is the syntax of an single line comment
+
+/*
+
```
Comment (hemanthreddy739) [inline demo_file.java:16]: try to import some packages

**Review by hemanthReddy125 — COMMENTED** [review]
hey hemanthreddy739 i made changes as u requested please check it again and if it is ok then merge

**Review by hemanthreddy739 — COMMENTED** [review]
ya is all right i'm merging

**Review by hemanthReddy125 — COMMENTED** [review]
thank you :)

**Review by hemanthreddy739 — APPROVED** [review]
ya it i merged check status

**Review by hemanthReddy125 — COMMENTED** [review]
hey, could u delete the branch also :)

**Comment by hemanthreddy739** [comment]
done :)

**Comment by hemanthReddy125** [comment]
hey thanks man

**Comment by hemanthreddy739** [comment]
u r welcoome

**Pull request merged**